website/docs/add-secure-apps/providers/rac/rac-public-key.md

```diff
@@ -0,0 +1,31 @@
+---
+title: RAC SSH Public Key Authentication
+---
+
+## About RAC SSH public key authentication
+
+The RAC provider supports SSH public key authentication. This allows for secure connections to SSH endpoints without the use of username and passwords.
+
+## Current limitations
+
+Each RAC provider is limited to a single private key. This means that a RAC provider with a private key set, will utilize that private key for all connections.
```

Comment:
That's incorrect, the private key can also be set on a property mapping and in the endpoint settings and can be fetched from user attributes